cont3xt/vueapp/src/components/links/LinkGroupCard.vue

```diff
@@ -464,54 +526,26 @@ export default {
           try {
             options = JSON.parse(options);
           } catch (e) {
-            /* eslint-disable no-template-curly-in-string */
-            console.error('getUrl: array requires valid JSON for parameters. Placeholder should look like this:\n${array,{"iType":"ip","include":"top","sep":"OR","quote":"\\""}}');
-            console.error('trying to parse', options);
+            console.error('Error trying to parse', options);
             console.error(e);
+            // remove the placeholder so the url isn't wonky
             url = url.substring(0, begin - match[0].length) + url.substring(end + 1);
             return url;
           }
 
-          if (!options.iType) {
-            console.error('getUrl: array requires an iType');
-            url = url.substring(0, begin - match[0].length) + url.substring(end + 1);
-            return url;
-          }
-
-          options.include ??= 'all'; // default to all
-          options.quote ??= ''; // default to no quote
-          options.sep ??= ','; // default to comma
-
-          if (['domain', 'ip', 'url', 'email', 'hash', 'phone', 'text'].indexOf(options.iType) === -1) {
-            console.error('getUrl: array requires a valid iType');
-            url = url.substring(0, begin - match[0].length) + url.substring(end + 1);
-            return url;
-          }
-
-          const indicators = [];
-          for (const key in this.$store.state.indicatorGraph) {
-            const indicator = this.$store.state.indicatorGraph[key];
-            if (indicator.indicator.itype === options.iType) {
-              if (options.include === 'all' ||
-                (options.include === 'top' && (indicator.parentIds.has(undefined) || indicator.parentIds.size === 0))) {
-                indicators.push(indicator.indicator.query);
-              }
-            }
-          }
-
-          if (indicators.length) {
-            const indicatorStr = options.quote + indicators.join(`${options.quote}${options.sep}${options.quote}`) + options.quote;
-            url = url.substring(0, begin - match[0].length) + indicatorStr + url.substring(end + 1);
-          } else {
-            url = url.substring(0, begin - match[0].length) + url.substring(end + 1);
+          if (match[1] === 'array') { // the type of match - array|start|end
```

Comment:
if only the language supported some way to not need lots of if statments

Reply:
why do you hate callbacks?

Reply:
They hurt me 🔪 😆

Reply:
well when we add the next elseif you can learn your lesson ;)